python/paddle/fluid/tests/book/test_fit_a_line.py

```diff
@@ -84,7 +84,7 @@ def train_loop(main_program):
                                           feed=feeder.feed(data),
                                           fetch_list=[avg_cost])
                 if avg_loss_value[0] < 10.0 or pure_bf16:
-                    if save_dirname is not None and not pure_bf16:
+                    if save_dirname is not None:
                         fluid.io.save_inference_model(save_dirname, ['x'],
```

Comment:
Maybe we could use paddle.static.save_inference here, fluid.io.save_inference look like deprecated. What is your opinion on that?

Reply:
I agree that it is better to test the up-to-date functions. I changed it.